manager/integration/tests/run.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 if [[ $@ =~ "--junitxml=" ]] ; then 
-  flake8
+  flake8 >> /dev/null
 
-  pytest -v $@  >> /tmp/longhorn-pytest 2>&1
+  pytest -v $@  > /tmp/longhorn-pytest
```

Comment:
The output of the testing pod will be used as the JUnit report, so that means it's impossible for now to see any progress or output during running tests via `pytest`, correct?

If yes, we should improve this part later to let people know what's going during the test instead of waiting for the report generated (nightly build will take hours to finish).

Reply:
Yes we need to wait to the test to finish to get the result.
will try figure something out for it later.